app/models/external_registries/verloren_of_gevonden_bike.rb

```diff
@@ -0,0 +1,89 @@
+module ExternalRegistries
+  class VerlorenOfGevondenBike < ExternalRegistryBike
```

Comment:
The issue with this not being STI is that there isn't a way to get back to this class from an instantiated `ExternalRegistryBike`.

If instead, this was STI, when you instantiated the `ExternalRegistryBike`, you would get back a `VerlorenOfGevondenBike`.

As of now, this class only has the `build_from_api_response`, which might be enough.

But what if we want to add a method to `VerlorenOfGevondenBike`s, not just another database attribute? It would free us up to have `external_registry_url` be a method rather than in the database.

Reply:
Gotcha, thanks for elaborating. STI would not have occurred to me for this use case, so the intuition isn't quite there — the detail is helpful.

> As of now, this class only has the `build_from_api_response`

I'm not sure I see a way around a builder method since the ActiveRecord constructor can't accept arbitrary keys (and overriding it does not seem desirable) and we need to accommodate varying schema from different api response objects, but I'm open to suggestions.

Reply:
Updated in 817062aeac9